app/helpers/patient_details_helper.rb

```diff
@@ -203,8 +207,6 @@ def custom_export_details_for_export
       pause_notifications: pause_notifications || false,
       expected_purge_ts: expected_purge_date_exp || '',
       monitoring_reason: monitoring_reason || '',
-      follow_up_reason: follow_up_reason || '',
-      follow_up_note: follow_up_note || '',
```

Comment:
Why this removal?

Reply:
this was simply moved here: https://github.com/SaraAlert/SaraAlert/pull/1006/files#diff-6a782ad659c7d687b8b004c762706d9c4c1ee24674305f8df71babe430abdb73R76

Reply:
Shouldn't it be in both places? One is for verifying linelist export and the other is for custom export.

Reply:
it gets combined here: https://github.com/SaraAlert/SaraAlert/pull/1006/files#diff-6a782ad659c7d687b8b004c762706d9c4c1ee24674305f8df71babe430abdb73R226

Reply:
Ohh i was confused because your first link was to the linelist fields not the full history fields, but I see they are there as well 👍